deb-x64/Dockerfile

```diff
@@ -24,6 +24,10 @@ ARG RUSTUP_VERSION=1.24.3
 ARG RUSTUP_SHA256="3dc5ef50861ee18657f9db2eeb7392f9c2a6c95c90ab41e45ab4ca71476b4338"
 ARG PATCHELF_VERSION=0.13.1
 ARG PATCHELF_SHA256="f6d5ecdb51ad78e963233cfde15020f9eebc9d9c7c747aaed54ce39c284ad019"
+ARG BISON_VERSION="3.8"
+ARG BISON_SHA256="d5d184d421aee15603939973a6b0f372f908edfb24c5bc740697497021ad9458"
+ARG BINUTILS_VERSION="2.39"
+ARG BINUTILS_SHA256="d12ea6f239f1ffe3533ea11ad6e224ffcb89eb5d01bbea589e9158780fa11f10"
```

Comment:
nit: Not a blocker but this block of `ARG`s is really hard to visually parse quickly - we may want to maybe use the hanging defs to make it easier to read:
```dockerfile
ARG FOO="3.2.1" \
    FOO_SHA="abc123"
ARG BAR="1.2.3" \
    BAR_SHA="abc123"
```

Reply:
This already took this form; I'd rather address additional changes to the dockerfile in a separate PR that might address maintainability issues.